README.md

```diff
@@ -299,3 +298,39 @@ NOTE: GenQuery2 column names are subject to change as the implementation is impr
 | TICKET_ALLOWED_USER_TICKET_ID | R_TICKET_ALLOWED_USERS | ticket_id |
 | TICKET_ALLOWED_GROUP_NAME | R_TICKET_ALLOWED_GROUPS | group_name |
 | TICKET_ALLOWED_GROUP_TICKET_ID | R_TICKET_ALLOWED_GROUPS | ticket_id |
+
+## Questions and Answers
+
+### Can I use the GenQuery2 microservices with the Python Rule Engine Plugin (PREP)?
+
+Yes! The important thing to remember is that the GenQuery2 microservices can update input arguments. This behavior is fully supported by the iRODS Rule Language, but not the PREP. Changes to input arguments are only visible via the object returned from the microservice.
```

Comment:
I notice an exclamation point - perhaps a bit much - but okay to leave it if you feel strongly.

Reply:
I don't have any attachment to it.

Will remove.

Reply:
Done.